### ISSUE TITLE
graphql-ez/plugin-upload using with pothos

### ISSUE THREAD

**Issue by 524c** [PabloSzx/graphql-ez#913]
Hello everyone, I would like some help with the following case:

Today I have a graphql api, and I have the following components:
Fastify as web framework and Pothos to build schemas.

I'm trying to integrate graphql-ez/plugin-upload into my ecosystem, but I'm having trouble creating a scalar Upload using pothos. Has anyone used graphql-ez/plugin-upload with pothos? can anyone help me?

**Comment by PabloSzx** [comment]
could you create a small repo with what you are trying to accomplish? thank you

**Comment by 524c** [comment]
> could you create a small repo with what you are trying to accomplish? thank you

hello, thank you very much for replying, yes I will create a repo.

**Comment by 524c** [comment]
> could you create a small repo with what you are trying to accomplish? thank you

Hello, I created this repository with a minimal working code:
[](https://github.com/524c/grapql-ez-pothos-test/)

My goal is to replace the current graphql-upload module with your module and make it work with the pothos I use to define schema.

**Comment by 524c** [comment]
> could you create a small repo with what you are trying to accomplish? thank you

Hello, my friend!!

I managed to solve my issue and create a Scalar for the File type. Was lost at that point before 😆
Now that I've managed to create a pothos-compatible scalar I can go ahead and integrate with your module. Thank you very much for your attention and sorry for the noise.

<img width="629" alt="file-scalar" src="https://user-images.githubusercontent.com/527641/188282511-1e2b83cc-32b5-4e28-a459-41bcaf4da4bc.png">
<img width="741" alt="resolver-upload" src="https://user-images.githubusercontent.com/527641/188282515-0801317e-f0cc-46c2-b732-20e8ff4622cd.png">
<img width="707" alt="service-upload" src="https://user-images.githubusercontent.com/527641/188282517-0bae34f7-5d71-43a4-95f8-c41c67877caa.png">
